src/content-page-repository/transformDocumentLink.ts

```diff
@@ -0,0 +1,40 @@
+import fs from 'fs';
+import path from 'path';
+import util from 'util';
+import { LinkProps } from '../model/LinkProps';
+import type { TransformationOptions } from './TransformationOptions';
+
+const copyFile = util.promisify(fs.copyFile);
+
+export async function transformDocumentLink(
+  link: LinkProps,
+  pagePath: string,
+  options: TransformationOptions,
+): Promise<LinkProps> {
+  const href = link?.href || '';
```

Comment:
если href пуст, то здесь нечего делать
```suggestion
  if (!link?.href) {
    return link;
  }
```

Reply:
++